filestack/utils/upload_utils.py

```diff
@@ -101,7 +106,8 @@ def multipart_upload(apikey, filepath, storage, upload_processes=None, params=No
         'filename': filename,
         'mimetype': mimetype,
         'size': filesize,
-        'store_location': storage
+        'store_location': storage,
+        'workflows': workflows
```

Comment:
Would be good to have this tested.
Also, it should be enough to add workflows only to `multipart/complete` request

Reply:
Do you mean adding workflows=workflows exactly to the body of the request?

Reply:
There is a confusing part that happens if I just add double quotation to workflows ids and change params['workflows'] to the newly made ids. In this scenario webhooks do not show the result of workflows while using a post request to get the workflows result work. Using this way, both webhooks and post request show workflows results.